src/shared/credentials/credentialsFile.ts

```diff
@@ -91,27 +90,13 @@ async function loadCredentialsFile(
     return parseIni(await slurpFile(credentialsFilePath))
 }
 
-// TODO: FOR POC-DEMOS ONLY, NOT FOR PRODUCTION USE!
-// REMOVE_BEFORE_RELEASE
-// This is nowhere near resilient enough :-)
+// TODO: consider using ini NPM module to implement this
 export async function saveProfile(
```

Comment:
We're pretty aggressive about deleting unused code. You can just remove `saveProfile` instead of throwing an error.